python/ray/tune/progress_reporter.py

```diff
@@ -52,6 +57,21 @@
     IS_NOTEBOOK = False
 
 
+SKIP_RESULTS_IN_REPORT = {"config", TRIAL_ID, EXPERIMENT_TAG, DONE}
+
+SKIP_RLLIB_RESULTS_IN_REPORT = {
+    "hist_stats",
+    "sampler_results",
+    "sampler_perf",
+    "info",
+    "timers",
+    "perf",
+    "counters",
+    "episode_media",
+    "num_healthy_workers",
+}
```

Comment:
This should remain in the RLlib module

Reply:
Ah this is a leftover from a previous iteration, I'll remove this